### ISSUE TITLE
[BUG/ISSUE] Clock disappeared

### ISSUE THREAD

**Issue by LeTaquiner** [martinet101/ElevenClock#857]
## Before submitting the issue

- [x] **I have checked that the bug is not a duplicate or, if it is, it needs to be reopened**
- [x] **I have checked that this issue is not present in the [Known Issues](https://github.com/martinet101/ElevenClock/wiki#known-issues) section on the wiki.**
- [x] **I have checked that I am running the latest version of ElevenClock**
- [x] **I have tried resetting and reinstalling ElevenClock and the issue persisted or happened again**
- [x] **I have checked the [Common Problems](https://github.com/martinet101/ElevenClock/wiki#common-problems) section on the wiki for solutions**

</b>

## Describe the bug

Eleven o'clock has disappeared from the top-left corner, overnight,  except for when mousing over the auto-hidden Windows 11 toolbar.  The Windows 11 system-tray clock also disappeared at the same time.
Restarting did not fix it.

## To Reproduce
Steps to reproduce the behaviour:

## Expected behaviour
I expect the clock to always be visible, top-left where I have it configured for, when not running any full-screen applications.  I do _not_ have a full-screen application running on either of my logical desktops - despite what the log appears to indicate

## The Log
It took 0.0009970664978027344 to load all language files
---------------------------------------------------------------------------------------------------

   ElevenClock's 3.7.0 (v3.7) log: Select all the text and hit Ctrl+C to copy it
   All modules loaded successfully and sys.stdout patched correctly, starting main script
   Translator function set language to "en"

---------------------------------------------------------------------------------------------------

 Log legend:
 🔵: Verbose
 🟢: Information
 🟡: Warning
 🟠: Handled unexpected exception
 🔴: Unhandled unexpected exception
 🟣: Handled expected exception

🔵 Initializing clock 0...
🔵 Using taskbar background color
🟢 Regular sized taskbar
🔵 Monitor geometry: PySide2.QtCore.QRect(0, 0, 1920, 1080)
🟡 ElevenClock set to hide with the taskbar
🟡 Clock on the top
🟡 Clock on the left
🟢 Using qt's default positioning system
🔵 Clock geometry: PySide2.QtCore.QRect(4, 2, 200, 46)
🔵 Font families   : ['Segoe UI Variable Display', 'sans-serif']
🔵 Custom font     : 
🔵 Font size: -1.0
🟢 Using white text (dark mode)
🔵 tDateMode: M/d/yyyy
🔵 tTimeMode: HH:mm
🔵 Loaded date time format: %H:%M:%S 
🔵 Full screen rect:  (0, 0, 1920, 1080)
🟣 Expected AttributeError on backgroundLoop thread
🔵 Initializing clock 0...
🔵 Using taskbar background color
🔵 Show/hide loop started with parameters: HideonFS:True, NotHideOnTB:False, HideOnRDP:False, ClockOn1Mon:True, NefWSMethod:True, DisableNotifications:False, legacyFullScreenMethod:False
🟢 Regular sized taskbar
🔵 Monitor geometry: PySide2.QtCore.QRect(0, 0, 1920, 1080)
🟠 Clock is cover!!!
🟡 ElevenClock set to hide with the taskbar
🟡 Clock on the top
🟢 Clock on the right
🟢 Using qt's default positioning system
🔵 Clock cover geometry: PySide2.QtCore.QRect(1716, 1034, 200, 46)
🔵 Font families   : ['Segoe UI Variable Display', 'sans-serif']
🔵 Custom font     : 
🔵 Font size: -1.0
🔵 tDateMode: M/d/yyyy
🔵 tTimeMode: HH:mm
🔵 Loaded date time format: %H:%M:%S 
🔵 Full screen rect:  (0, 0, 1920, 1080)
🔵 Show/hide loop started with parameters: HideonFS:True, NotHideOnTB:True, HideOnRDP:True, ClockOn1Mon:True, NefWSMethod:True, DisableNotifications:True, legacyFullScreenMethod:False
can't parse ""
can't parse ""
🟢 Loaded clocks in 0.7743263244628906
🔵 Restart loop: 1
🟢 Cached text input host hwnd: 131296
🟢 Cached text input host hwnd: 131296
None
🟣 Expected AttributeError on backgroundLoop thread
None
🟣 Expected AttributeError on backgroundLoop thread
🔵 System tray icon enabled
🔵 Starting update check
🔵 Low cpu mode is set to False. DisableNotifications is set to True
🟢 Loaded everything in 1.8303132057189941
🟢 Green lignt to rease temp dirs
🔴 Traceback (most recent call last):

🔴   File ".\tools.py", line 246, in clearTmpDir

🔴   File "shutil.py", line 749, in rmtree

🔴   File "shutil.py", line 619, in _rmtree_unsafe

🔴   File "shutil.py", line 617, in _rmtree_unsafe

🔴 PermissionError: [WinError 5] Access is denied: 'C:\\Users\\kiwig\\AppData\\Local\\Temp\\_MEI207842\\libcrypto-1_1.dll'

🔴 Note this traceback was caught by reporter and has been added to the log ([WinError 5] Access is denied: 'C:\\Users\\kiwig\\AppData\\Local\\Temp\\_MEI207842\\libcrypto-1_1.dll')
🔵 Version URL: https://versions.somepythonthings.tk/versions/elevenclock.ver
🟢 Updates not found
🟡 Closing clock on {'Monitor': (0, 0, 1920, 1080), 'Work': (0, 0, 1920, 1080), 'Flags': 1, 'Device': '\\\\.\\DISPLAY1'}
🟡 Closing clock on {'Monitor': (0, 0, 1920, 1080), 'Work': (0, 0, 1920, 1080), 'Flags': 1, 'Device': '\\\\.\\DISPLAY1'}
🟡 Closing clock on {'Monitor': (0, 0, 1920, 1080), 'Work': (0, 0, 1920, 1080), 'Flags': 1, 'Device': '\\\\.\\DISPLAY1'}
🔵 Initializing clock 0...
🔵 Using taskbar background color
🟢 Regular sized taskbar
🔵 Monitor geometry: PySide2.QtCore.QRect(0, 0, 1920, 1080)
🟡 ElevenClock set to hide with the taskbar
🟡 Clock on the top
🟡 Clock on the left
🟢 Using qt's default positioning system
🔵 Clock geometry: PySide2.QtCore.QRect(4, 2, 200, 46)
🔵 Font families   : ['Segoe UI Variable Display', 'sans-serif']
🔵 Custom font     : 
🔵 Font size: -1.0
🟢 Using white text (dark mode)
🔵 tDateMode: M/d/yyyy
🔵 tTimeMode: HH:mm
🔵 Loaded date time format: %H:%M:%S 
🔵 Full screen rect:  (0, 0, 1920, 1080)
🔵 Initializing clock 0...
🔵 Using taskbar background color
🔵 Show/hide loop started with parameters: HideonFS:True, NotHideOnTB:False, HideOnRDP:False, ClockOn1Mon:True, NefWSMethod:True, DisableNotifications:False, legacyFullScreenMethod:False
🟢 Regular sized taskbar
🔵 Monitor geometry: PySide2.QtCore.QRect(0, 0, 1920, 1080)
🟠 Clock is cover!!!
🟡 ElevenClock set to hide with the taskbar
🟡 Clock on the top
🟢 Clock on the right
🟢 Using qt's default positioning system
🔵 Clock cover geometry: PySide2.QtCore.QRect(1716, 1034, 200, 46)
🔵 Font families   : ['Segoe UI Variable Display', 'sans-serif']
🔵 Custom font     : 
🔵 Font size: -1.0
🔵 tDateMode: M/d/yyyy
🔵 tTimeMode: HH:mm
🔵 Loaded date time format: %H:%M:%S 
🔵 Full screen rect:  (0, 0, 1920, 1080)
🔵 Show/hide loop started with parameters: HideonFS:True, NotHideOnTB:True, HideOnRDP:True, ClockOn1Mon:True, NefWSMethod:True, DisableNotifications:True, legacyFullScreenMethod:False
🔵 tDateMode: M/d/yyyy
🔵 tTimeMode: HH:mm
🔵 Loaded date time format: %H:%M:%S 
🟢 Cached text input host hwnd: 131296
🟢 Cached text input host hwnd: 131296
🔵 Restart loop: 1
🔵 Starting update check
🔵 Version URL: https://versions.somepythonthings.tk/versions/elevenclock.ver
🟢 Updates not found
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🟡 Fullscreen window detected! (0, 0, 1920, 1080) Fullscreen rect: (0, 0, 1920, 1080)
🔵 Starting update check
🔵 Version URL: https://versions.somepythonthings.tk/versions/elevenclock.ver
🟢 Updates not found
🔵 Starting update check
🔵 Version URL: https://versions.somepythonthings.tk/versions/elevenclock.ver
🟢 Updates not found
🔵 Announcement URL: https://www.somepythonthings.tk/resources/elevenclock.announcement
🔵 Image URL: https://images.vexels.com/media/users/3/128603/isolated/lists/c3baedd2e3624a2ed60434ab856cea3d-plantilla-de-venta-de-burbuja-roja.png
🔵 Copying log to the clipboard...
🟢 Log copied to the clipboard successfully!
🔵 Copying log to the clipboard...

```
Please paste here the log (Go to ElevenClock Settings -> Scroll to the bottom -> Open ElevenClock Log. 

If the issue can be reproduced (As an example: After waiting 5 minutes, the clock shows a wrong format), please 
send the log AFTER reproducing the issue (1. Launch ElevenClock, 2. Reproduce the issue, 3. Paste the log)
```

**Comment by LeTaquiner** [comment]
> Can you please disable updates and install version 3.6.4?

Done.  
ElevenClock and Win11 clock both immediately returned.

**Comment by martinet101** [comment]
The next version (3.7.1) should have this behaviour fixed. Please update and confirm if this works

**Comment by LeTaquiner** [comment]
> The next version (3.7.1) should have this behaviour fixed. Please update and confirm if this works

This fixed ElevenClock (i.e. it remained visible after updating 3.6.4 -> 3.7.1.) but killed the Win11 system tray clock/calendar.

**Comment by martinet101** [comment]
> > The next version (3.7.1) should have this behaviour fixed. Please update and confirm if this works
> 
> This fixed ElevenClock (i.e. it remained visible after updating 3.6.4 -> 3.7.1.) but killed the Win11 system tray clock/calendar.

can you please give more details?

**Comment by LeTaquiner** [comment]
> > > The next version (3.7.1) should have this behaviour fixed. Please update and confirm if this works
> > 
> > 
> > This fixed ElevenClock (i.e. it remained visible after updating 3.6.4 -> 3.7.1.) but killed the Win11 system tray clock/calendar.
> 
> can you please give more details?

Immediately upon installing the 3.7.1 update, ElevenClock came back to the top-left; but the Win11 clock/date in the lower-right corner on the taskbar, disappeared.   However, as I returned to my computer to reply - it's now there.  
I now have no known issues with 3.7.1.  Apologies for misdirection and thanks for your good works!

**Comment by LeTaquiner** [comment]
Sorry, I am a fool.  I was distracted by work, and forgot that an hour ago I had reinstated to 3.6.4 to test.

I have to reopen this case.

With 3.6.4. installed: I have ElevenClock top-left as I have it configured to be;  Win11 has clock/date-time/notification lower-right.  If I mouseover the Win11 clock, I see my second timezone clock.

With 3.7 installed: Neither ElevenClock nor Win 11 clocks are visible.  However, when mousing over the taskbar, ElevenClock came back.

With 3.7.1 installed: ElevenClock is perfect; but Win 11 clock/date is missing from lower-right system tray.  I watched it disappear as 3.7.1 got installed.  Mousing over it does nothing.
![image](https://user-images.githubusercontent.com/110486291/182604239-76f85ad5-099f-4f78-9bd5-fa507285d8db.png)
